tests/bson/bug0592.phpt

```diff
@@ -32,6 +32,7 @@ foreach ($tests as $json) {
 ===DONE===
 <?php exit(0); ?>
 --EXPECTF--
+%a
```

Comment:
I'll make the debug=stderr changes, but the other PR will just remove them again...